ar_exercise/exercise.rb

```diff
@@ -31,41 +34,91 @@ class Apartment < ActiveRecord::Base
+all_apartments = Apartment.where(address:"6005 Damien Corners")
+
 # get all tenants in that apartment
+all_tenants = Tenant.where(apartment_id: 6)
 
 # Use `each` and `puts` to:
 # Display the name and ID # of every tenant
+Tenant.all.each do |tenant|
+  puts "Name: #{tenant.name}, Id: #{tenant.id}"
+end
 # Iterate over each apartment, for each apartment, display it's address and rent price
+Apartment.all.each do |apartment|
+  puts "Address: #{apartment.address}, Rent: $#{apartment.monthly_rent}"
+end
 # Iterate over each apartment, for each apartment, display it's address and all of it's tenants
+Apartment.all.each do |apartment|
+  puts apartment.address
+  apartment.tenants.each do |tenant|
+    puts tenant.name
+  end
+  puts "-" * 50
+end
 
 ################################################
 # CREATING / UPDATING / DELETING
 ################################################
 
-# Hint, the following methods will help: `new`, `create`, `save`, `uddate`, `destroy`
+# Hint, the following methods will help: `new`, `create`, `save`, `update`, `destroy`
 
 # Create 3 new apartments, and save them to the DB
+the_bat_cave = Apartment.create(address: "123 Main St", monthly_rent: 2000, sqft: 600, num_beds: 2, num_baths: 1)
+# ID 22
+the_sinatra = Apartment.create(address: "1094 Bergen St", monthly_rent: 1800, sqft: 700, num_beds: 2, num_baths: 1)
+# ID 23
+the_wu_tang = Apartment.create(address: "2970 Flatbush Ave", monthly_rent: 1100, sqft: 650, num_beds: 1.5, num_baths: 1)
+# ID 24
+the_jeter = Apartment.create(address: "825 Fulton St", monthly_rent: 1300, sqft: 480, num_beds: 1, num_baths: 1)
+
 # Create at least 9 new tenants and save them to the DB. (Make sure they belong to an apartment)
+margaret = Tenant.create(name: "Margaret", age: 24, gender: "Female", apartment_id: 22)
+adam = Tenant.create(name: "Adam", age: 30, gender: "Male", apartment_id: 24)
+benedict = Tenant.create(name: "Benedict", age: 85, gender: "Male", apartment_id: 22)
+georgia = Tenant.create(name: "Georgia", age: 49, gender: "Female", apartment_id: 24)
+olu = Tenant.create(name: "Olu", age: 28, gender: "Male", apartment_id: 24)
+benson = Tenant.create(name: "Benson", age: 41, gender: "Male", apartment_id: 23)
+shukura = Tenant.create(name: "Shukura", age: 29, gender: "Female", apartment_id: 22)
+ross = Tenant.create(name: "Ross", age: 28, gender: "Male", apartment_id: 23)
+cassandra = Tenant.create(name: "Cassandra", age: 26, gender: "Female", apartment_id: 23)
+
 # Note: you'll use this little bit of code as a `seeds.rb` file later on.
 
 # Birthday!
 # It's Kristin Wisoky's birthday. Find her in the DB and change her age to be 1 year older
 # Note: She's in the seed data, so she should be in your DB
+Tenant.find_by(name: "Kristin Wisoky").increment(:age)
 
 # Rennovation!
 # Find the apartment "62897 Verna Walk" and update it to have an additional bedroom
 # Make sure to save the results to your database
+apartment = Apartment.where(address:"62897 Verna Walk")
```

Comment:
This step finds the right apartment, but 2 things:

1. I'd suggest using `find_by` instead of where when you know it's *one* result
2. You still need to update the bedroom